src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,48 +1,74 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  userId: number;
+  todo: Todo;
+  fnSelectTodo: (selectTodoId: number) => void;
+};
+
+export const TodoModal: React.FC<Props> = (
+  {
+    userId,
+    todo,
+    fnSelectTodo,
+  },
+) => {
+  const [user, setUser] = useState<User | null>(null);
+
+  useEffect(() => {
```

Comment:
You should include the `userId` in the dependency array of `useEffect` to ensure the user is fetched again if the `userId` changes. 🔄